src/Device/Driver/AR62xx.cpp

```diff
@@ -256,7 +256,7 @@ uint16_t AR62xxDevice::FrequencyToAR62xxId(const RadioFrequency freq)
   unsigned frequency_offset = frequency - MIN_FREQUENCY;
 
   /** generate frequency-id */
-  uint16_t frequency_id = std::round(frequency_offset * MAX_FREQUENCY_ID / FREQUENCY_RANGE);
+  uint16_t frequency_id = frequency_offset * MAX_FREQUENCY_ID / FREQUENCY_RANGE;
```

Comment:
This was rounding previously (even though it was buggy and the truncation already happened before). If rounding is what you want, then this change is wrong.

Reply:
no, rounding is not needed anymore as this was based on previous code of eku60 which was based on decimals (originally the simply added 0.5 which was rounding-like... anyway it was buggy, but now it's working fine with RadioFrequency)
tested with live-system